### PR TITLE
add mpd(Music Player Daemon) plugin

### PR DESCRIPTION
**Pull request by mqus**
Same as in https://github.com/albertlauncher/python/pull/56 .

This enables the mpd prefix, which will search the mpd database for song/album/artist names and allow the user to add one or multiple songs to the current mpd playlist.

To change the controlled instance, one has to edit the SERVER and PORT variables.

It needs the python package [mpd2](https://github.com/Mic92/python-mpd2)

### CODE REVIEW

File: mpd/__init__.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+# Link: https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/ 
+
+# Work which could still be done:
+#   - mpd commands (e.g. play/pause, stop, repeat, random, skip, etcpp)
+#   - search by path/path-completion
+#   - more ways to start/add a song
+#   - search playlists
+
+def handleQuery(query):
+
```
Comment (hulloanson) [inline mpd/__init__.py]: Check if query is triggered.

File: mpd/__init__.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+    if len(query.string)>1:
+        # set icons:
+        song_icon=iconLookup("audio-x-generic") or os.path.join(os.path.dirname(__file__),"Papirus-Team-Papirus-Mimetypes-Audio-x-flac.svg")
+        album_icon=iconLookup("music-album-default") or os.path.join(os.path.dirname(__file__),"folder-green-music.svg")
+        artist_icon=iconLookup("music-artist-default") or os.path.join(os.path.dirname(__file__),"folder-green-image-people.svg")
+        
+        mpclient = MPDClient()
+        mpclient.timeout = 10
+        mpclient.connect(SERVER,PORT)
```
Comment (hulloanson) [inline mpd/__init__.py]: What if the connection fails? You should show a message for it.

**Comment by mqus** [comment]
Added an error message and the trigger-check.

**Comment by ManuelSchneid3r** [comment]
Sorry for letting you wait so long.

Please send the PR to the Python repo. The community approach did not work out.

Further I changed the policy considering plugins. If you want to be the active maintainer (own repo, own issuetracker, etc.) of the plugin, please place it in a separate repository and make this repo a git submodule of the Python plugin repo. If you do not want to be a maintainer, well, you can still send a PR to the Python plugin repo (no submodule in this case), but don't expect a merge. It depends on the demand and maintenance workload the plugin has.

**Pull request closed without merging**